src/index.js

```diff
@@ -49,6 +15,30 @@ function opts(file, config) {
 
 exports.interfaceVersion = 2;
 
+function getPlugins(file, target) {
+    const OptionManager = require('babel-core/lib/transformation/file/options/option-manager'); // eslint-disable-line
```

Comment:
Looking at this again, you can actually use `require('babel-core').OptionManager`.